### ISSUE TITLE
SAVE / READ User preferences

### ISSUE THREAD

**Issue by zeevox** [EAZY-Code/sound-recorder#1]
Save preferences on pressing [the close button](https://github.com/EZ-Code-Developer/sound-recorder/blob/master/app/src/main/java/com/ezcode/recorder/SettingsActivity.java#L46)
Read previous preferences from SharedPreferences when [opening SettingsActivity](https://github.com/EZ-Code-Developer/sound-recorder/blob/master/app/src/main/java/com/ezcode/recorder/SettingsActivity.java#L37)

**Comment by egkoppel** [comment]
Settings are automatically read on open if you used settings activity template.

**Comment by zeevox** [comment]
The Settings Template was not used in this case.

**Comment by egkoppel** [comment]
However, to read you can use this code
```Java
SharedPreferences sharedPref = PreferenceManager.getDefaultSharedPreferences(this);
String syncConnPref = sharedPref.getString("key", "value if non existent");
```

**Comment by egkoppel** [comment]
In 2.0 stable the settings seem to persist

**Comment by zeevox** [comment]
> @egkoppel In 2.0 stable the settings seem to persist
True, a bug that is now magically fixed in v2.0.
